src/Kafka/KafkaInstanceDrawer/KafkaConnectionTab.tsx

```diff
@@ -0,0 +1,134 @@
+import {
+  TextContent,
+  Text,
+  TextVariants,
+  Skeleton,
+  ClipboardCopy,
+  Button,
+  Label,
+  Popover,
+  ButtonVariant,
+} from "@patternfly/react-core";
+import { HelpIcon } from "@patternfly/react-icons";
+import { FunctionComponent } from "react";
+import { useTranslation } from "react-i18next";
+import { Link } from "react-router-dom";
+import "./KafkaInstanceDrawer.css";
+
+export type KafkaConnectionTabProps = {
+  isKafkaPending?: boolean;
+  externalServer?: string;
+  tokenEndPointUrl: string;
+  linkToServiceAccount: string;
+  linkToAccessTab: string;
+  showCreateServiceAccountModal: () => void;
+};
+
+export const KafkaConnectionTab: FunctionComponent<KafkaConnectionTabProps> = ({
+  isKafkaPending,
+  externalServer,
+  tokenEndPointUrl,
+  linkToServiceAccount,
+  linkToAccessTab,
+  showCreateServiceAccountModal,
+}) => {
+  const { t } = useTranslation();
+
+  return (
+    <div className="mas--details__drawer--tab-content">
+      <TextContent className="pf-u-pb-sm">
+        <Text component={TextVariants.small}>
+          {t("common:drawer_resource_tab_body_description_1")}
+        </Text>
+        <Text component={TextVariants.h3} className="pf-u-mt-lg">
+          {t("common:bootstrap_server")}
+        </Text>
+        <Text component={TextVariants.small}>
+          {t("kafka:connection_tab.bootstrap_server_description")}
+        </Text>
+        {isKafkaPending ? (
+          <Skeleton fontSize="2xl" />
+        ) : (
+          <ClipboardCopy
+            data-testid="drawerStreams-copyBootstrapURL"
+            textAriaLabel={t("common:bootstrap_server")}
+          >
+            {externalServer}
+          </ClipboardCopy>
+        )}
+      </TextContent>
+      <TextContent className="pf-u-pb-sm">
+        <Text component={TextVariants.h3} className="pf-u-mt-xl">
+          {t("service-account:service_accounts_small")}
+        </Text>
+        <Text component={TextVariants.small}>
+          {t("service-account:create_service_account_to_generate_credentials")}{" "}
+          <Link to={linkToServiceAccount} data-testid="tableStreams-linkKafka">
+            {t("service-account:service_accounts")}
+          </Link>{" "}
+          {t("common:page")}.
```

Comment:
This structure is problematic for translation. When we do eventually send strings for translation, we want to send complete strings, like 
> Create a service account to generate credentials. Manage the service accounts you create on the [Service Accounts](https://619e1921b3e38b003afeb2be-lwmiwzufiw.chromatic.com/to-service-account) Page.

But with the way the strings are broken up here, a translator would receive these as separate strings:

- > Create a service account to generate credentials. Manage the service accounts you create on the
- > Service Accounts
- > page.

This is likely to result in poor translations. 

I'm not super familiar with the translation package that we're using. Is it possible to use a structure like this one: https://github.com/redhat-developer/app-services-ui-components/blob/main/locales/en/common.json#L25
e.g. `"Create a service account to generate credentials. Manage the service accounts you create on the {{pageName}} page."` with `"Service Accounts"` defined as a separate string that's passed in for `{{pageName}}`?